Formula/c/couchdb.rb

```diff
@@ -46,24 +38,16 @@ class Couchdb < Formula
   end
 
   def install
-    spidermonkey = Formula["spidermonkey@91"]
-    inreplace "configure", '[ ! -d "/usr/local/include/${SM_HEADERS}" ]',
-                           "[ ! -d \"#{spidermonkey.opt_include}/${SM_HEADERS}\" ]"
-    inreplace "src/couch/rebar.config.script" do |s|
-      s.gsub! "-I/usr/local/include/mozjs", "-I#{spidermonkey.opt_include}/mozjs"
-      s.gsub! "-L/usr/local/lib", "-L#{spidermonkey.opt_lib} -L#{HOMEBREW_PREFIX}/lib"
-    end
-
-    system "./configure", "--spidermonkey-version", spidermonkey.version.major.to_s
+    system "./configure", "--disable-spidermonkey", "--js-engine=quickjs"
```

Comment:
No `quickjs` dependency?

Reply:
It didn't seem possible to unbundle.

`quickjs` is at least designed to be embedded and only builds static libs so there is less of an advantage as dependency.